doc/tutorials/using_partition_mode_on_up2.rst

```diff
@@ -187,9 +187,8 @@ Enabling partition mode
   .. code-block:: console
      :emphasize-lines: 2
 
-      ACRN Partition Mode with Zephyr
-     *ACRN Partition Mode
-      ADG Partition Mode
+     *Ubuntu
```

Comment:
OK, I just made a text version of the askew screen photo that was there.  You need to change it to ``:emphasis-lines: 1`` above since it's now the first line of the block.